pkg/manager/member/tiflash_member_manager.go

```diff
@@ -629,6 +629,11 @@ func (m *tiflashMemberManager) syncTidbClusterStatus(tc *v1alpha1.TidbCluster, s
 	// This only returns Up/Down/Offline stores
 	storesInfo, err := pdCli.GetStores()
 	if err != nil {
+		if pdapi.IsTiKVNotBootstrappedError(err) {
```

Comment:
Can we still keep the original logic for TiFlash?

Reply:
OK